packages/web-components/src/card/card.ts

```diff
@@ -0,0 +1,207 @@
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public interactive: boolean = false;
+
+  /**
+   * @property disabled
+   * @default false
+   * @remarks
+   * Determines disabled state of card
+   */
+  @attr({ mode: 'boolean' })
+  public disabled: boolean = false;
+
+  /**
+   * @property selected
+   * @default false
+   * @remarks
+   * Determines selected state of card
+   */
+  @observable
+  @attr({ mode: 'boolean' })
+  public selected: boolean = false;
+
+  /**
+   * @method sizeChanged
+   * @remarks
+   * Updates the computed stylesheet when the size of the card changes
+   */
+  public sizeChanged(prev: boolean | undefined, next: boolean): void {
+    this.updateComputedStylesheet();
+  }
+
+  /**
+   * @method selectedChanged
+   * @remarks
+   * Emits an event when the selected state of the card changes
+   */
+  public selectedChanged = (): void => {
+    this.$emit('onSelectionChanged', this.selected);
+  };
+
+  /**
+   * @method interactiveChanged
+   * @remarks
+   * Updates the focusable state of the card when the interactive state changes
+   */
+  public interactiveChanged = (): void => {
+    this.setInteractive();
+  };
+
+  /**
+   * @method setFocusable
+   * @remarks
+   * Sets the tabindex attribute based on the interactive state of the card
+   * @internal
+   */
+  public setInteractive(): void {
+    if (this.interactive) {
+      this.setAttribute('tabindex', '0');
+      this.setAttribute('aria-selected', this.selected ? 'true' : 'false');
+    } else {
+      this.removeAttribute('tabindex');
+      this.removeAttribute('aria-selected');
+    }
+  }
+
+  /**
+   * Updates an internal stylesheet with calculated CSS custom properties.
+   *
+   * @internal
+   */
+  protected updateComputedStylesheet(): void {
+    // Determine the pixel value based on the controlSize attribute
+    let sizeValue;
+    let borderRadiusValue;
+    switch (this.controlSize) {
+      case CardControlSize.small:
+        sizeValue = '8px';
+        borderRadiusValue = '2px';
+        break;
+      case CardControlSize.medium:
+        sizeValue = '12px';
+        borderRadiusValue = '4px';
+        break;
+      case CardControlSize.large:
+        sizeValue = '16px';
+        borderRadiusValue = '8px';
+        break;
+      default:
+        sizeValue = '12px';
+        borderRadiusValue = '4px';
+        break;
+    }
+
+    this.$fastController.removeStyles(this.computedStylesheet);
+
+    this.computedStylesheet = css`
+      :host {
+        --card--size: ${sizeValue};
+        --card--border-radius: ${borderRadiusValue};
+      }
+    `;
+
+    this.$fastController.addStyles(this.computedStylesheet);
+  }
+
+  /**
+   * Handle the checked state of the Card when interactive
+   *
+   * @param e - the mouse event
+   * @internal
+   */
+  public clickHandler(e: MouseEvent): boolean | void {
+    if (!this.disabled && this.interactive) {
+      this.selected = !this.selected;
+    }
+  }
```

Comment:
This seems to assume that the card will always be "uncontrolled", is there a method for someone to set and manage the selected state themselves? Events are a common paradigm on the web, should this component have an event? If we'd like it to be both _controlled_ and _uncontrolled_, we'll need to do more work to ensure both scenarios are managed.

Reply:
- Added onSelectionChanged event
- Added explicit methods for selecting/unselecting the card